docs/source/_static/bibliography.bib

```diff
@@ -1,162 +1,195 @@
-@article{Cotter2015,
-  title={On the shallow atmosphere approximation in finite element dynamical cores},
-  author={Cotter, CJ and Ham, DA and McRae, ATT and Mitchell, L and Natale, A},
-  url={http://arxiv.org/abs/1410.3069},
-  archivePrefix ="arXiv",
-  eprint =       {1410.3069},
-  year =         {2015}
+@InProceedings{Bokhove2016,
+  author =       {Onno Bokhove and Anna Kalogirou},
+  title =        {Variational water wave modelling: from continuum to
+                  experiment},
+  booktitle =    {Lectures on the theory of water waves},
+  year =         2016,
+  editor =       {Thomas J. Bridges and Mark D. Groves and David
+                  P. Nicholls},
+  volume =       426,
+  series =       {LMS Lecture Note Series},
+  pages =        {226--260},
+  publisher =    {Cambridge University Press},
+  url =
+                  {http://www1.maths.leeds.ac.uk/~matak/documents/lms-cup2015.pdf}
 }
 
-@article{Cotter2015a,
+@Article{Cotter2014,
+  author =       {Colin J. Cotter and Robert C. Kirby},
+  title =        {Mixed finite elements for global tide models},
+  journal =      {Numerische Mathematik},
+  year =         2015,
+  pages =        {1--23},
+  doi =          {10.1007/s00211-015-0748-z},
+  url =          {http://arxiv.org/abs/1410.0045}
+}
+
+@Article{Cotter2015,
+  author =       {Colin J. Cotter and David A. Ham and Andrew
+                  T. T. McRae and Lawrence Mitchell and Andrea Natale},
+  title =        {On the shallow atmosphere approximation in finite
+                  element dynamical cores},
+  year =         2015,
+  url =          {http://arxiv.org/abs/1410.3069},
+  archiveprefix ={arXiv},
+  eprint =       {1410.3069}
+}
+
+@Article{Cotter2015a,
+  author =       {Colin J. Cotter and Dmitri Kuzmin},
   title =        {Embedded discontinuous Galerkin transport schemes
                   with localised limiters},
-  author =       {Cotter, CJ and Kuzmin, D},
-  archivePrefix ="arXiv",
+  year =         2015,
+  archiveprefix ={arXiv},
   eprint =       {1509.04431},
-  url =          {http://arxiv.org/abs/1509.04431},
-  year =         {2015}
+  url =          {http://arxiv.org/abs/1509.04431}
 }
 
-@Article{Homolya2015,
-  author = 	 {Mikl\`os Homolya and David A. Ham},
-  title = 	 {A parallel edge orientation algorithm for quadrilateral meshes},
-  journal = 	 {Submitted to SIAM Journal on Scientific Computing},
-  year = 	 2015,
-  archivePrefix ="arXiv",
-  eprint={1505.03357},
-  url = {http://arxiv.org/abs/1505.03357}
+@Article{Homolya2016,
+  author =       {Mikl\`os Homolya and David A. Ham},
```

Comment:
Ugh, wrong accent. The correct one is the _acute accent_, not the _grave accent_.

    \` -> \'